pkgs/top-level/all-packages.nix

```diff
@@ -24022,6 +24022,8 @@ in
 
   insync = callPackage ../applications/networking/insync { };
 
+  insync-v3 = libsForQt5.callPackage ../applications/networking/insync/v3.nix { };
```

Comment:
You can change the 5 here to 515 or 514 or some other version in the future to pin a QT version.

Reply:
Great idea, done.